freestyle-doobie/src/main/scala/doobie.scala

```diff
@@ -16,7 +16,7 @@
 
 package freestyle
 
-import cats.{~>, Foldable}
+import cats.Foldable
```

Comment:
It seems we aren't using `Foldable` in this file, so we can remove this as well.